acsys/dpm/__init__.py

```diff
@@ -287,157 +366,97 @@ def _xlat_reply(self, msg):
                             micros=msg.micros)
         return msg
 
-    def __aiter__(self):
-        return self
-
-    async def __anext__(self):
-        while True:
-            try:
-                if len(self._qrpy) > 0:
-                    return self._qrpy.pop(0)
-
-                _, msg = await self.gen.__anext__()
-                msg = self._xlat_reply(msg)
-
-                # If the message is not None, return it. If it is
-                # None, start at the top of the loop.
-
-                if msg is not None:
-                    return msg
-                continue
-
-            except acsys.status.Status as exception:
-
-                # If we're disconnected from ACNET, re-throw the
-                # exception because there's more work to be done to
-                # restore the state of the program.
-
-                if exception == acsys.status.ACNET_DISCONNECTED:
-                    raise
-
-            except GeneratorExit:
-                pass
-
-            # If we've reached here, DPM returned a fatal ACNET
-            # status. Whatever it was, we need to pick another DPM and
-            # add all the current requests.
-
-            _log.warning('DPM(id: %s) connection closed ... retrying',
-                         str(self.list_id))
-            await self._restore_state()
+    async def connect(self):
+        loop = asyncio.get_event_loop()
+        con_fut = loop.create_connection(lambda: self,
+                                         host='dce46.fnal.gov',
```

Comment:
Just a reminder that we need to change this to `acsys-proxy` before merging.

Reply:
I added it to the checklist in the pull request.